core/themes/bartik/templates/comment.tpl.php

```diff
@@ -72,9 +75,11 @@
       <span class="new"><?php print $new; ?></span>
     <?php endif; ?>
 
-    <?php print render($title_prefix); ?>
-    <h3><?php print $title; ?></h3>
-    <?php print render($title_suffix); ?>
+    <?php if ($title_display == TRUE): ?>
```

Comment:
This could be shortened (doesnt have to be so, just saying) to 
`<?php if ($title_display): ?>`